site/testsuites/testsuite-javascript/__tests__/tbdex/wallet/quickstart.test.js

```
@@ -0,0 +1,239 @@
+        // :snippet-start: walletQuickstartProcessQuote
+        // Wait for Quote message to appear in the exchange
+        exchangeId = rfq.exchangeId;
+        while (!quote) {
+            try {
+                const exchange = await TbdexHttpClient.getExchange({
+                    pfiDid: pfiDid,
+                    did: customerDid,
+                    exchangeId: exchangeId
+                });
+
+                quote = exchange.find(msg => msg instanceof Quote);
+
+                if (!quote) {
+                    // Make sure the exchange is still open
+                    close = exchange.find(msg => msg instanceof Close);
+                    
+                    if(close) { break; } 
+                    else {
+                    // Wait 2 seconds before making another request
+                    await new Promise(resolve => setTimeout(resolve, 2000));
+                    }
+                }
+            } catch (e) {
+                // do nothing to let the loop keep running in the event
+                // the exchange hasn't populated yet
+            }
+        }
+        // :snippet-end:
+
+        expect(quote).toBeDefined();
+    });
+
+    test('Create Order', async () => {
+        while (!order) {
+            try {
+                // :snippet-start: walletQuickstartCreateOrder
+                order = Order.create({
+                    metadata: {
+                    from: customerDid.uri,         // Customer's DID
+                    to: pfiDid,                    // PFI's DID
+                    exchangeId: exchangeId,        // Exchange ID from the Quote
+                    protocol: "1.0"                // Version of tbDEX protocol you're using
+                    }
+                });
+                // :snippet-end:
+
+                expect(order).toBeDefined();
+            } catch (e) {
+                // do nothing to let the loop keep running in the event
```

ignore an exception?